CHANGES/8300.bugfix

```diff
@@ -0,0 +1 @@
+Fix rate limit feature
```

Comment:
Can this be more specific? Maybe something like "Fixed bug where rate limiting wasn't being used."